### ISSUE TITLE
coin sendto testing

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#114]
I apologize for I've tested a command you've mentioned [here](https://github.com/slimcoin-project/pacli/issues/74#issuecomment-2035743380) as not urgent to test, but I was too curious to test it because I felt it would be very useful for me personally to use instead of the slimcoind native command.
So I've tested `coin sendto mmjaHuS1kTRMFJaTJqYMjdmF18n36eXHfD 1` from the address `mixcqdAy4aFtuBV7uRebqBTjiFWCcS4UEp` that reports to have 27856.93 coins and got the following error message:
```
File "~/.local/bin/pacli", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/__main__.py", line 479, in main
    fire.Fire({
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 143, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 477, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
                                ^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 693, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/coin.py", line 26, in sendto
    raise RecieverAmountMismatch
pypeerassets.exceptions.RecieverAmountMismatch
```
Don't know if it's relevant though.

**Comment by d5000** [comment]
This is one of the original vanilla commands taking the amount and address in brackets (see help). I don't think I will modify it much more.

**Comment by d5000** [comment]
> By the way is there any limit of the addresses I can send the coins to with this command?

I don't think so, at least in the code there isn't a limit. You may however run into two problems: 1) the fee may be too low if your transaction is over 1 kB and it won't get confirmed, and 2) there may be an internal total limit in Slimcoin for the transaction size (in Bitcoin it's 100 kB).

Brackets are now no longer mandatory as of commit 343afa2 if you have only one address and one amount. I also edited the help somewhat.

See also issue #327. I think this can be closed now.
